client/state/inline-help/selectors.js

```diff
@@ -1,31 +1,27 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import { get } from 'lodash';
+
 /**
  * Returns the current search query.
  *
  * @param  {Object}  state  Global state tree
  * @return {String}        The current search query
  */
 export function getSearchQuery( state ) {
-	return state.inlineHelpSearchResults.search.searchQuery || '';
+	return get( state, 'inlineHelpSearchResults.search.searchQuery', '' );
 }
 
 /**
  * Returns the index of the currently selected search result.
  *
  * @param  {Object}  state  Global state tree
- * @return {String}        The index of the currently selected search result
+ * @return {Integer}        The index of the currently selected search result
  */
 export function getSelectedResult( state ) {
```

Comment:
Looking at this now I believe we should rename this to `getSelectedResultIndex`. Thoughts?